### ISSUE TITLE
I think the use of 't_label' is forbidden for unsupervised learning tasks when training.

### ISSUE THREAD

**Issue by chihuadelishu** [Solacex/Domain-Consensus-Clustering#9]
Why do you use the labels of target domain (i.e. t_label) to calculate loss during training models?I think the use of 't_label' is forbidden for unsupervised learning tasks.
PS:  .\trainer\dcc1_trainer.py:
class Trainer(BaseTrainer):
    def iter(self, i_iter):
      ......
       en_loss = self.memory.forward(t_feat, t_label, self.config.t, False)

**Comment by Solacex** [comment]
Hello,
We did not use the ground truth of target samples, the t_label you mentioned is the pseudo label that obtained through clustering. For details, you can refer to 

https://github.com/Solacex/Domain-Consensus-Clustering/blob/5a626e1a7b294ed8fa71205bbc549a93b566e864/dataset/__init__.py#L107

**Comment by chihuadelishu** [comment]
Thanks for your prompt reply. It is my negligence. :)
